openedx/features/edly/admin.py

```diff
@@ -49,9 +49,25 @@ class EdlyOrganizationAdmin(admin.ModelAdmin):
     """
     Admin interface for the EdlyOrganization object.
     """
+    search_fields = ['name', 'slug']
     list_display = ['name', 'slug', 'created', 'modified']
     inlines = [EdlySubOrganizationInlineAdmin]
 
 
+class EdlyUserProfileAdmin(admin.ModelAdmin):
+    """
+    Admin interface for the EdlyUserProfile object.
+    """
+    search_fields = ['edly_sub_organization__name']
+    list_display = ['user_username', 'edly_sub_organizations_name']
+
+    def user_username(self, obj):
+        return obj.user.username
+
+    def edly_sub_organizations_name(self, obj):
```

Comment:
Names can be of 255 characters each we need to only show slugs
```
edly_sub_organizations_slugs
```

Reply:
Done